static-site/src/components/splash/index.jsx

```diff
@@ -54,86 +53,144 @@ class Splash extends React.Component {
+                Nextstrain groups represent collections of datasets, potentially with controlled access.
+              </Styles.CenteredFocusParagraph>
+            </FlexCenter>
+            <FlexCenter>
+              <Cards
+                squashed
+                compactColumns
+                cards={createGroupCards([{name: "neherlab"}, {name: "spheres"}])}
+              />
+            </FlexCenter>
+            <BigSpacer/>
+            <FlexCenter>
+              <Styles.Button to="/groups">
+                See all groups
+              </Styles.Button>
+            </FlexCenter>
+          </div>
+        </div>
 
         <HugeSpacer/>
+        <div className="row">
+          {/* PATHOGENS */}
+          <div className="col-md-6">
+            <ScrollableAnchor id={'pathogens'}>
+              <Styles.H1>Explore pathogens</Styles.H1>
+            </ScrollableAnchor>
+            <FlexCenter>
+              <Styles.CenteredFocusParagraph>
+                Genomic analyses of specific pathogens kept up-to-date by the Nextstrain team.
+              </Styles.CenteredFocusParagraph>
+            </FlexCenter>
+            <FlexCenter>
+              <Cards
+                squashed
+                compactColumns
+                cards={coreCards}
+              />
+            </FlexCenter>
+            <BigSpacer/>
+            <FlexCenter>
+              <Styles.Button to="/pathogens">
+                See all pathogens
+              </Styles.Button>
+            </FlexCenter>
+          </div>
 
-        <ScrollableAnchor id={'community'}>
-          <Styles.H1>From the community</Styles.H1>
-        </ScrollableAnchor>
-
-        <FlexCenter>
-          <Styles.CenteredFocusParagraph>
-            Analyses by independent groups <a href="https://docs.nextstrain.org/en/latest/guides/share/community-builds.html">stored and
-            accessed via public GitHub repos</a>
-          </Styles.CenteredFocusParagraph>
-        </FlexCenter>
-
-        <Cards
-          cards={communityCards}
-        />
-
+          {/* COMMUNITY */}
+          <div className="col-md-6">
+            <ScrollableAnchor id={'community'}>
+              <Styles.H1>From the community</Styles.H1>
+            </ScrollableAnchor>
+            <FlexCenter>
+              <Styles.CenteredFocusParagraph>
+                Analyses by independent groups <a href="https://docs.nextstrain.org/en/latest/guides/share/community-builds.html">stored and
+                accessed via public GitHub repos</a>.
+              </Styles.CenteredFocusParagraph>
+            </FlexCenter>
+            <FlexCenter>
+              <Cards
+                squashed
+                compactColumns
+                cards={communityCards}
+              />
+            </FlexCenter>
+            <BigSpacer/>
+            <FlexCenter>
+              <Styles.Button to="/community">
+                See featured community analyses
```

Comment:
Perhaps "Learn more" is more appropriate, while we don't actually have any featured analyses on that page!

Reply:
resolved in 8919324